lib/setup.js

```diff
@@ -209,9 +221,10 @@ async function setup() {
   const defaultPython = isWindows() ? "python" : "python3";
   const python = getInput("python", /^.+$/, defaultPython);
   const useInstaller = core.getBooleanInput("use-installer");
+  const token = core.getInput("token", { required: false });
```

Comment:
Should we provide a default like `version` and `python` input?

Reply:
updated in the new revision.